ggml/src/ggml-cuda/norm.cu

```diff
@@ -157,6 +181,11 @@ static __global__ void rms_norm_f32(
         if constexpr (do_multiply) {
             const int mul_col = col % mul_ncols;
             dst[col] = scale * x[col] * mul[mul_col];
+
+            if constexpr (do_add) {
+                const int add_col = col % add_ncols;
+                dst[col] += add[add_col];
+            }
```

Comment:
Either add a `static_assert` to ensure that the case `do_add && !do_multiply` never happens or change the implementation to be correct for it.